src/enforcer.js

```diff
@@ -358,27 +408,31 @@ export class TrustedTypesEnforcer {
    * @private
    */
   getKey_(object, name) {
+    // TODO(msamuel): Can we use Object.prototype.toString.call(object)
```

Comment:
Is it worth it? This is to part of the private install-time API to Proxy the original setters. `object` here is a prototype of various `Element`s. If those are spoofed at the moment of running this code, we're Proxying fake DOM anyway.

Reply:
IIUC, they are also used at restore time which happens after untrusted inputs reach the system.

I don't think it's a huge issue.
I mention it because it provides a reliable path to facts like
```js
Object.prototype.toString.call(Boolean.prototype) === '[object Boolean]'
Object.prototype.toString.call(Function.prototype) === '[object Function]'
Object.prototype.toString.call(Number.prototype) === '[object Number]'
// etc.
```

but not much moreso than typeof except for Array/Object distinctions and a few intrinsics like `Date`.

Reply:
I see. We can add it then (though install/uninstall are not intended to be called at all, those methods are used in tests only, to be able to reuse the document object). Though, to be fair, I should have just created a new Document object in `beforeEach` or documented the intention of those methods.

Reply:
I can leave this for now, or file a tracking issue.